_includes/get_a_quote.html

```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
         We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
     </p>
     <p>
-        When you hire us, we will have a trial period of a month, after which we will evaluate our working relationship, to ensure that you are happy with the work we are doing for you.
+        If you get in touch, we will first arrange a meeting to talk about your concrete business situation.
```

Comment:
I think that we should ship the 'if' part and make the sentence more direct and 'active'.
Something like:

Get in touch, and get a (free) meeting to talk about your concrete business situation.